spec.html

```diff
@@ -116,7 +116,7 @@ <h1>WrappedFunctionCreate ( _callerRealm_, _targetFunction_ )</h1>
 			1. Assert: IsCallable(_targetFunction_) is *true*.
 			1. Let _internalSlotsList_ be the internal slots listed in <emu-xref href="#table-internal-slots-of-wrapped-function-exotic-objects"></emu-xref>, plus [[Prototype]] and [[Extensible]].
 			1. Let _obj_ be ! MakeBasicObject(_internalSlotsList_).
-			1. Set _obj_.[[Prototype]] to %Function.prototype% from _callerRealm_.
+			1. Set _obj_.[[Prototype]] to _callerRealm_.[[Intrinsics]].[[%Function.prototype%]].
```

Comment:
i'm not sure this is notation we have anywhere - intrinsics aren't internal slots to my understanding.

Reply:
@ljharb I believe they are.

A Realm will have the [`[[Intrinsics]]` field](https://tc39.es/ecma262/#sec-code-realms), and CreateRealm eventually [populates it with fields](https://tc39.es/ecma262/#sec-createintrinsics) of _intrinsics_ where the names are those listed in the column one of table 8 (Intrinsics).

Reply:
Fair enough - altho the intrinsic notation is the value, so this might need to be a string?

Reply:
Maybe `_callerRealm_.[[Intrinsics]].%Function.prototype%`? The intrinsic is not exactly an internal of `[[Intrinsics]]`?

Reply:
Bare `%` notation refers to a value, so it doesn't make sense after a `.` nor inside `[[ ]]`. https://tc39.es/ecma262/#sec-createintrinsics says "the name of", which implies a string, so perhaps:
```suggestion
			1. Set _obj_.[[Prototype]] to _callerRealm_.[[Intrinsics]].[[*"%Function.prototype%"*]].
```

Reply:
pinging @tc39/ecma262-editors

Reply:
I accepted the suggestion. We use the name as a string in ecma-262, but I can't find any example accessing it from the `[[Intrinsics]]` so this might open precedent.